docs/src/components/MainSectionSubsection.js

```diff
@@ -10,15 +11,37 @@ type Props = {|
 |};
 
 const MainSectionSubsection = ({ children, description, title }: Props): Node => {
+  const slugifiedId = slugify(title || '');
   return (
     <>
       {title && (
-        <Box paddingY={1}>
-          <Heading size="sm">{title}</Heading>
+        <Box
+          dangerouslySetInlineStyle={{
+            __style: {
+              scrollMarginTop: 60,
+            },
+          }}
+          id={slugifiedId}
+          data-anchor
+          paddingY={2}
+        >
+          <Flex alignItems="baseline" gap={2}>
+            <Heading size="sm">{title}</Heading>
+            <IconButton
+              dangerouslySetSvgPath={{
+                __path:
```

Comment:
are we reusing this link icon somewhere else? could we create a const and export if so? Maybe it's just used here...

Reply:
Will address this separately